<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathWindow</class>
 <widget class="QMainWindow" name="PathWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1366</width>
     <height>768</height>
    </size>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>768</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1366</width>
      <height>768</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_sus">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>270</y>
      <width>131</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushOkButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>580</y>
      <width>105</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textSus">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>230</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic; color:#ffffff;&quot;&gt;Suspeitos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushInicioButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>580</y>
      <width>105</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Inicio</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textLugar">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic; color:#ffffff;&quot;&gt;Escolha um lugar para ir:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBackButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>580</y>
      <width>105</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>371</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <item>
     <property name="text">
      <string>Pra√ßa central</string>
     </property>
     <property name="font">
      <font>
       <family>Iosevka NF</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="HorPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/Images/menu.jpeg</normalon>
      </iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mercado central</string>
     </property>
     <property name="font">
      <font>
       <family>Iosevka NF</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="HorPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/Images/wallpaper1.jpeg</normalon>
      </iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Estalagem</string>
     </property>
     <property name="font">
      <font>
       <family>Iosevka NF</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="HorPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/Images/castle.jpg</normalon>
      </iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Corte do Rei</string>
     </property>
     <property name="font">
      <font>
       <family>Iosevka NF</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="HorPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/Images/wolf.jpg</normalon>
      </iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Enfermagem</string>
     </property>
     <property name="font">
      <font>
       <family>Iosevka NF</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="HorPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/Images/j4uakmaghau61.jpg</normalon>
      </iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Casa do Duque</string>
     </property>
     <property name="font">
      <font>
       <family>Iosevka NF</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="HorPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/Images/taverna.jpg</normalon>
      </iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Biblioteca Central</string>
     </property>
     <property name="font">
      <font>
       <family>Iosevka NF</family>
       <pointsize>15</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="HorPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Taverna</string>
     </property>
     <property name="font">
      <font>
       <family>Iosevka NF</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="HorPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Entrada da Capital</string>
     </property>
     <property name="font">
      <font>
       <family>Iosevka NF</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="HorPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bairro dos moradores</string>
     </property>
     <property name="font">
      <font>
       <family>Iosevka NF</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="HorPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Parte Baixa da Cidade</string>
     </property>
     <property name="font">
      <font>
       <family>Iosevka NF</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="HorPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <zorder>label</zorder>
   <zorder>pushBackButton</zorder>
   <zorder>textLugar</zorder>
   <zorder>pushOkButton</zorder>
   <zorder>textSus</zorder>
   <zorder>pushInicioButton</zorder>
   <zorder>comboBox_sus</zorder>
   <zorder>listWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
